src/LuaTransformer.ts

```
@@ -758,7 +771,7 @@ export class LuaTransformer {
         const hasStaticSetters = tsHelper.hasSetAccessorInClassOrAncestor(statement, true, this.checker);
 
         if (extendsType) {
-            const extendedTypeNode = tsHelper.getExtendedTypeNode(statement, this.checker);
+            const extendedTypeNode = tsHelper.getExtendedTypeNode(statement, this.checker)!;
```

Should we check this result and throw if it's undefined?